samples/matter/light_bulb/sample.yaml

```diff
@@ -61,3 +61,10 @@ tests:
       - nrf52840dk_nrf52840
       - nrf5340dk_nrf5340_cpuapp
       - nrf7002dk_nrf5340_cpuapp
+  sample.matter.light_bulb.internal.smp_dfu:
+    build_only: true
+    extra_args: CONFIG_CHIP_DFU_OVER_BT_SMP=y FILE_SUFFIX=release_internal
```

Comment:
I don't really like `release_internal` name, because it applies only to the one board in a single scenario and sounds like some internal release of an application. If this configuration makes DFU work on nRF54L, wouldn't `nrf54l_dfu` be better?

Reply:
or even nrf54l_internal_dfu. I know that it is not a good name, but I will propose something different.